patching/src/main/java/org/jboss/as/patching/runner/AbstractPatchingTask.java

```diff
@@ -152,7 +152,7 @@ public void execute(final PatchingTaskContext context) throws IOException {
             return;
         }
         final PatchContentLoader contentLoader = description.getLoader();
-        final boolean skip = skipExecution | context.isExcluded(contentItem);
+        final boolean skip = skipExecution || context.isExcluded(contentItem);
```

Comment:
This one could have been the more tricky to understand the possible original intention, but there the logic executed on the right side method is idempotent and it doesn't change the internal behavior, so it should be fine for the change.